commands.js

```diff
@@ -1,22 +1,95 @@
-const { sendEmbedMessage } = require('./helpers.js');
+const { sendEmbedMessage, trim } = require('./helpers.js');
+const axios = require('axios');
 
 module.exports = {
   matchCommand: function (Discord, config, logger, message, command, args) {
     switch(command) {
       case 'help':
         commandHelp(Discord, config, logger, message, command, args);
         break;
+      case 'ud':
+        commandUrbanDictionary(Discord, config, logger, message, command, args);
+        break;
       default:
         message.channel.send(`That command is not one I know yet - but you could add it! To find out more visit ${config.github_url}`);
     }
   },
 };
 
 function commandHelp (Discord, config, logger, message, command, args) {
-  const msg = `
+  const helpMessage = `
 The commands available to you are:
-- ${config.prefix}commandgoeshere   - Do something
+\`${config.prefix}ud <query>\`   - Search Urban Dictionary
+\`${config.prefix}help <command>\` - For help with a specific command add the command as an argument
   `;
 
-  message.channel.send(msg);
+  // Specify specific command help
+  if (args.length > 0) {
+    switch (args[0]) {
+      case 'ud':
+        message.channel.send(`Usage: \`${config.prefix}ud <query>\`    *Example: ${config.prefix}ud yeet*`);
+        break;
+      default:
+        message.channel.send(`That command is not one I know yet - but you could add it! To find out more visit ${config.github_url}`);
+    }
+  }
+  else {
+    message.channel.send(helpMessage);
+  }
+
+
+}
+
+function commandUrbanDictionary(Discord, config, logger, message, command, args) {
+  if (!args.length) {
+    //If the command is used incorrectly without arguments
+    return message.channel.send(`You didn't provide any arguments, ${message.author}!\nCorrect Usage: \`${config.prefix}ud <query>\``);
+  }
+  else if (args[0] != "") {
+    var searchQuery = args[0];
```

Comment:
We're using ES6 syntax, so any vars need to be replaced with let or const, depending on whether or not the value of it will be changed after the initial declaration.

Reply:
Bad habits die hard.